### ISSUE TITLE
Accessibility - 2.1.1 Keyboard - Critical

### ISSUE THREAD

**Issue by phucduong86** [shiffman/shiffman.net#84]
In mobile mode, user cannot access navigation toggle button by keyboard. Recommend implementing toggle button as focusable element.

**Comment by phucduong86** [comment]
Resolved by PR #99
